coroutines/build.gradle.kts

```diff
@@ -0,0 +1,47 @@
+plugins {
+    kotlin("multiplatform")
+}
```

Comment:
not sure if anything else is needed here for publishing this as a separate artifact.